front/src/app/scenes/transaction/transaction.component.ts

```diff
@@ -63,6 +70,46 @@ export class TransactionComponent implements OnInit, OnDestroy {
     this._layoutService.offLoading();
   }
 
+  processTransaction() {
+    if (!this.tx.parsedLogs.length) {
+      return;
+    }
+    forkJoin([
+      this._commonService.eventsAbi$,
+      this._commonService.getAddress(this.tx.to),
+    ]).subscribe(([events, address]: [ContractEventsAbi, Address]) => {
+      this.tx.addressData = address;
+      this.tx.processedLogs = this.tx.parsedLogs.map((log: TxLog) => {
+        const prettyLog: ProcessedLog = new ProcessedLog();
+        prettyLog.index = +log.logIndex;
+        prettyLog.contract_address = log.address;
+        prettyLog.removed = log.removed;
+        let abiItem: AbiItem;
+        const eventSignature = <string>log.topics[0];
```

Comment:
Probably need to account for logs with no topics and not index-out-of-bounds here.